exp/metadataserver/proto/api/metadata_server.proto

```diff
@@ -0,0 +1,41 @@
+syntax = "proto3";
+
+// NOTE run `buf generate` from `exp/metadataserver/proto` to regenerate code
+package weaviate.internal.metadataserver;
+
+service MetadataService {
+  // QuerierStream is experimental, may be changed/removed. A QuerierStream represents a
+  // connection between this metadata cluster and a querier node. See the implementing
+  // function's doc for more information.
+  rpc QuerierStream(stream QuerierStreamRequest) returns (stream QuerierStreamResponse) {}
+}
+
+// QuerierStreamRequest is experimental, may be changed/removed. QuerierStreamRequest is currently
+// unused as we only care about the existence of the stream to register the querier node. We'll
+// likely add new types to this message in the future and use them.
+message QuerierStreamRequest {
+  enum Type {
+    TYPE_UNSPECIFIED = 0;
+  }
+  Type type = 1;
```

Comment:
@kavirajk was raising a very good point about `NxM` broadcast. Could a follow up PR (so that we can get this one through) improve the request to have a querier node be able to specify "which" tenant/class ?

Edit: Thinking about it actually it's not ideal, cause we want to have *a* node at least have the tenant to HOT, but not all of them. And then route request to it. 

@kavirajk I think we will tackle the NxM problem once we have routing in place, we'll be able to have the metadata server manage which node has which tenant locally, and ensure that *they* receive only the status updates for these tenants. 

For now I'm fine keeping it as a broadcast, but we'll have to tackle the problem in short order, before we release it to prod.

Reply:
Yep. Adding a note about the NxM problem and may be creating issue on our milestone would be nice. Feel free to skip in this PR 👍

Reply:
added here under R&D https://github.com/weaviate/weaviate-serverless/issues/92